build.gradle

```diff
@@ -161,7 +161,7 @@ wpi.java.configureExecutableTasks(jar)
 wpi.java.configureTestTasks(test)
 
 // Create version file
-project.compileJava.dependsOn(createVersionFile)
```

Comment:
I think it would be best to keep compileJava and compile kotlin since we have a mixed codebase.

Reply:
Good point. 
446ac8b